chemprop/cli/train.py

```diff
@@ -145,6 +153,10 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         default=0,
         help="Freeze the first ``n`` layers of the FFN from the checkpoint model (specified by ``--checkpoint``). The message passing layer should also be frozen with ``--freeze-encoder``.",
     )
+    transfer_args.add_argument(
+        "--from-foundation",
+        help=f"Name of pretrained foundation model used to initialize message passing. One of: {', '.join((FoundationModels.keys()))}",
```

Comment:
```suggestion
        help=f"Name of pretrained foundation model used to initialize message passing. One of: {', '.join((FoundationModels.keys()))}, or a path to a local model file.",
```
This could be updated to let users know they can pass their own model.